docs/tutorial/part-one/index.md

```diff
@@ -1,390 +1,282 @@
+`gatsby new [SITE_DIRECTORY] [URL_OF_STARTER_GITHUB_REPO]`
 
-Node.js is a programming tool for running JavaScript on servers and in your
-computer's terminal. Gatsby is built using Node.js.
+> 💡 See a list of the existing [official and community starters](/docs/gatsby-starters/)!
 
-Open a terminal window. See
-[terminal instructions for Mac users](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/) and
-[terminal instructions for Windows users](https://www.quora.com/How-do-I-open-terminal-in-windows). In your terminal window, type `node --version` and hit ENTER, then `npm --version` and hit ENTER (tip: to run a specified command, you must type the command into your terminal and then press ENTER. Then the command will run).
+If you omit a URL from the end, Gatsby will automatically generate a site for you based on the [default starter](https://github.com/gatsbyjs/gatsby-starter-default). Let’s do that now, to create a new site with the default starter, and explore the output.
 
-You should see something like:
+### ✋ Generate a new site based on Gatsby’s default starter
 
-![Check if node.js/npm is installed](check-versions.png)
+1. Open up your terminal. 
+2. Run `gatsby new my-tutorial-site`.
+3. Run `cd my-tutorial-site`.
+4. Run `gatsby develop`.
 
-Gatsby supports versions of Node back to v6 and npm to v3.
+This process should look familiar — it’s almost exactly what we did in the last section to scaffold out the "hello world" site. But this time, because we’re using a different starter, when you visit [http://localhost:8000](http://localhost:8000/) things will look a little different:
 
-If you don't have Node.js installed, go to https://nodejs.org/ and install the
-recommended version for your operating system.
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_E618395E4841D470F48D0AACF52A3EBAEA74345E0FB4AC285CBC937727755E20_1520719644125_Screen+Shot+2018-03-07+at+11.00.28+PM.png)
 
-## Install the "Hello World" starter
 
-Gatsby uses "starters" for starting new projects. Starters
-are partially built Gatsby sites that are pre-configured to help you get moving faster.
-There are several official starters and many others contributed from the Gatsby
-community! [See the Starters page for the full list](/docs/gatsby-starters/).
+Let’s take a look at the code we’ve just generated.
 
-To install a starter, first install Gatsby's command line program by running the following:
+## Gatsby’s project structure
+### ✋ Open up the code using your editor of choice.
 
-```sh
-npm install --global gatsby-cli
-```
+You should see something like this:
 
-Once that's installed, open a new terminal window and run the following commands to create a new Gatsby site in a directory called `tutorial-part-one` and then move to this new directory:
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_E618395E4841D470F48D0AACF52A3EBAEA74345E0FB4AC285CBC937727755E20_1522382930687_Screen+Shot+2018-03-29+at+11.06.11+PM.png)
 
```

Comment:
Ok, I love these gifs, but when going through it I realized we need code examples that are static. It was easy to miss what I was supposed to type and have to wait for the GIF to start over. I think GIFs are useful as an illustration of the visual effects of hot reloading.

Reply:
Interesting feedback -- the gifs are intended to be supplementary (they don't include any content that isn't stated statically beforehand). The gifs are nice, but perhaps it would be more helpful to have a static image, and just a code block of commands. Then use gifs more selectively, like you said.

Reply:
Code blocks (which will eventually have copy buttons) will be crucial. It's too easy to miss things when watching a GIF!

And yes, using them selectively is awesome. They really do show off very cool parts about coding with Gatsby :)